src/Servers/Kestrel/Core/src/KestrelServerLimits.cs

```diff
@@ -42,18 +42,26 @@ public class KestrelServerLimits
     private long? _maxConcurrentConnections;
     private long? _maxConcurrentUpgradedConnections;
 
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
```

Comment:
Can these be removed with the `P:` trick?

Reply:
I'm guessing the problem here is `SocketTransportOptions.MaxWriteBufferSize` and `QuicTransportOptions.MaxWriteBufferSize` can't be referenced because the Kestrel.Core project doesn't have references to the transports.

I suggest you use the `<c>` tag, e.g. `<c>SocketTransportOptions.MaxWriteBufferSize</c>` instead and remove the pragma warning disable.

Reply:
Removed.